src/components/calcite-pick-list/resources.ts

```diff
@@ -7,3 +7,6 @@ export enum ICON_TYPES {
   square = "square",
   grip = "grip"
 }
+export const TEXT = {
+  filterPlaceholder: "filter results"
```

Comment:
Should `filter` be capitalized?

Reply:
I prefer camelCase over PascalCase personally, and that's what we've been using. You feel strongly?

Reply:
I think he's referring to the "filter results".

Reply:
`filterPlaceholder: "Filter results"`